src/packages/property-editors/collection/config/bulk-action-permissions/permissions.element.ts

```diff
@@ -76,7 +76,7 @@ export class UmbPropertyEditorUICollectionPermissionsElement
 			<uui-toggle
 				?checked=${this.value.allowBulkCopy}
 				@change=${(e: UUIBooleanInputEvent) => this.#onChange(e, 'allowBulkCopy')}
-				label="Allow bulk copy (content only)"></uui-toggle>
+				label="Allow bulk duplicate (content only)"></uui-toggle>
```

Comment:
I was surprised to learn we don't have localization keys for these. I guess that's a good up-for-grabs task :)

Reply:
Good point. 👍